protocol_rfcs/managed-commits.md

```diff
@@ -144,16 +171,21 @@ interface CommitStore {
         actions: Iterator[String]): CommitResponse
 
     /**
-     * API to get the un-backfilled commits for the table represented by the given `tablePath` after
-     * the given `startVersion`. The returned commits are contiguous and in ascending version order.
+     * API to get the un-backfilled commits for the table represented by the given `tablePath` where
+     * `startVersion` <= version <= endVersion.
+     * If endVersion is -1, then it means that we want to get all the commits starting from `startVersion`
```

Comment:
Option[Long] seems better than magic values?

Reply:
Isn't Option more scalaish? I kept it -1 to more reflect it like a pseudo code.
No strong opinion - I can change it to Option if thats more intutive.